config/sidekiq.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'sidekiq'
+require 'sidekiq/api'
```

Comment:
Move this require into `app.rb`, since that's the only play we use the `Sidekiq::API`.